x-pack/plugins/session_view/common/mocks/responses/session_view_io_events.mock.ts

```diff
@@ -200,7 +200,7 @@ export const sessionViewIOEventsMock: ProcessEventResults = {
           id: '1',
         },
         process: {
-          entity_id: '2',
+          entity_id: '6',
```

Comment:
this should stay 2, as it's still the same process e.g vim

my PR to change the query logic has fixes to these ids.